tests/upgrade/lib.sh

```diff
@@ -60,7 +60,7 @@ validate_upgrade() {
     local upgrade_cluster_id="$3"
     local policies_dir="../pkg/defaults/policies/files"
 
-    if [ -n $API_TOKEN ]; then
+    if [[ -n ${API_TOKEN:-} ]]; then
```

Comment:
I think you need quotes around it too.
"${API_TOKEN:-}"

Reply:
yes